### PR TITLE
Changes to support commas as well as dots

### PR DESCRIPTION
**Pull request by ryanmitchell**

### CODE REVIEW

File: assets/js/cartbox.modal.js [hunk 1]
```diff
@@ -85,10 +85,11 @@
 		
 		price *= inputEl.val();
 		
-		var decimals = $cartItem.data('priceFormat').split('.').pop();
+		var decimalSeperator = $cartItem.data('priceFormat').indexOf('.') > -1 ? '.' : ',';
+		var decimals = $cartItem.data('priceFormat').split(decimalSeperator).pop();
```
Comment (sampoyigi) [inline assets/js/cartbox.modal.js:89]: How about using the format returned from `currency()->getFormatter()`?

**Review by sampoyigi — CHANGES_REQUESTED** [review]

File: assets/js/cartbox.modal.js (hunk 1 above)
Comment (sampoyigi) [reply assets/js/cartbox.modal.js:89]: Actually should be `currency()->getCurrency()->getFormat()`

Comment (ryanmitchell) [reply assets/js/cartbox.modal.js:89]: Not sure what you mean - we already get and pass the format as a data attribute?

Comment (sampoyigi) [reply assets/js/cartbox.modal.js:89]: Ah, you're right! Never mind then 🤦

**Pull request merged**